### PR TITLE
Delete old rcran images

### PR DESCRIPTION
**Pull request by rosbo**
- Delete images w/o the latest tag.
- Delete images older than 3 months.
- Delete images in `kaggle-images` and `kkb-infra` projects.

http://b/208859194

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -8,6 +8,17 @@ steps:
...
+# Delete old images
+- id: 'delete-old-images'
+  name: 'gcr.io/cloud-builders/gcloud'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    set -e
+    gcloud container images list-tags gcr.io/$PROJECT_ID/rcran-build --filter="NOT tags:latest timestamp.datetime < -P3M" --format='get(digest)' --limit 100 | xargs -I {} gcloud container images delete gcr.io/$PROJECT_ID/rcran-build@{} --quiet --force-delete-tags
+    gcloud container images list-tags gcr.io/kaggle-images/rcran --filter="NOT tags:latest timestamp.datetime < -P3M" --format='get(digest)' --limit 100 | xargs -I {} gcloud container images delete gcr.io/kaggle-images/rcran@{} --quiet --force-delete-tags
```
Comment (Philmod) [inline cloudbuild.yaml:20]: Should we add the GPU images too?

Comment (Philmod) [reply cloudbuild.yaml:20]: Why 3 months instead of 9 in the other projects?

**Review by Philmod — APPROVED** [review]

File: cloudbuild.yaml (hunk 1 above)
Comment (rosbo) [reply cloudbuild.yaml:20]: The `rcran` image doesn't have a GPU version. The GPU specific packages are installed in `rstats`.

Why 3 months, somewhat arbitrary. I figured we rarely debug the `rcran` image directly. I could be wrong. I can set it to 9 months. Or something else. WDYT?

Comment (rosbo) [reply cloudbuild.yaml:20]: I will use 6 months everywhere.

**Pull request merged**